contents/tutorials/analyze-surveys-with-chatgpt.md

```diff
@@ -0,0 +1,385 @@
+---
+title: How to analyze surveys with ChatGPT
+date: 2023-10-18
+author: ["lior-neu-ner"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-14.png
+tags: ['surveys']
+---
+
+[Surveys](/docs/surveys) are a great way of collecting feedback from your users, especially if you ask your users like "How can we improve our product?". However, they can be hard to analyze if you receive hundreds or more answers. Fortunately, [OpenAI's ChatGPT](https://openai.com/chatgpt) is great at doing this.
+
+In this tutorial, we'll show you how to use ChatGPT to analyze your survey results. We'll create a basic Node.js script, parse a CSV of survey answers, and use the ChatGPT API to extract useful information such as sentiment and theme.
+
+> The completed code for this tutorial is available on [GitHub](https://github.com/PostHog/analyze-surveys-with-chatgpt)
+
+## Creating a Node script
+
+First, make sure [Node is installed](https://nodejs.dev/en/learn/how-to-install-nodejs/) (18.16.0 or newer). Then create a `package.json` file by running `npm init`. Give your app a name and select the defaults for all the options.
+
+Then, create a new JavaScript file for our script (we name ours `survey-analyzer.js`):
+
+```bash
+npm init
+touch survey-analyzer.js
+```
+
+Finally, add a placeholder function to your script. We'll add code to it in the next steps.
+
+```js
+// in survey-analyzer.js
+
+const analyzeSurveyAnswers = async () => {
+  // TODO: add code
+}
+
+analyzeSurveyAnswers()
+```
+
+## Parsing a CSV of survey answers
```

Comment:
It would be really useful to know how to get a CSV from a PostHog survey. It is a bit confusing using a test CSV when you mention surveys to start.

Reply:
Added!